pkg/certificate/manager.go

```diff
@@ -99,7 +101,7 @@ func (m *Manager) EnsureCA(name, cn string) error {
 }
 
 // EnsureCertificate creates the specified certificate if it does not already exist
-func (m *Manager) EnsureCertificate(certReq Request, ownerID int) (Certificate, error) {
+func (m *Manager) EnsureCertificate(certReq Request, ownerID int, expiry time.Duration) (Certificate, error) {
```

Comment:
Maybe a simple unit test would be good, à la create CA, create cert, read in CA and cert and verify the right expiration days and key usages and so on.

Reply:
Done